### ISSUE TITLE
Readability issues

### ISSUE THREAD

**Issue by EmiMidnight** [ValentinTT/ida-debugger#3]
Hey there,
theme looks very cozy but there's at least two spots that are pretty bad contrast wise currently.

Lines with active breakpoints are completely unreadable on all 4 color variants:
![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/739810b7-d99f-41d4-a41e-26acc6187c50)

Clicking on a highlighted line/variable in the structure view is not completely unreadable, but still bad contrast wise I think:
![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/4527a2f2-8780-4ee5-854c-5cba2396d4cb)

Otherwise, everything else looks great over here. :)

**Comment by autumncpp** [comment]
Agreed. Breakpoints should probably use `color-red` with some added transparency.

Here are the properties that would require changing:
```css
qproperty-line-bg-bpt-enabled: ${color-teal}; /* switch to color-red and add transparency */

/* i am unsure what these properties would look like, maybe add transparency to them as well? */
qproperty-line-bg-bpt-disabled: ${color-text};
qproperty-line-bg-bpt-unavailable: ${color-overlay0};
```
The variable highlight is causing readability issues because it's actually layering it's transparency with the current-line highlight. Would probably be better to switch to using a different color entirely for variable highlights. I'll put a PR together for this.

**Comment by autumncpp** [comment]
I added more transparency to the cursor line background color and switched the highlight background color to maroon, as well as adding more transparency.

![](https://i.invalid.gg/ida64_sYXwROvOXO.png)

I also think this looks much better for breakpoints (red with 30% transparency):
![](https://i.invalid.gg/ida64_Dhthff0V4D.png)

**Comment by EmiMidnight** [comment]
Looks great! 🙏

I noticed one more issue, in the type declaration editor, the offset is invisible.
![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/c4494e5e-5d4c-4a52-a5f1-19fb5635f910)

I added 
```css
TextEdit QWidget {
    background-color: transparent;
} 
```

and that seems to have fixed it. 
Not sure if that causes any other changes I didn't notice.

**Comment by 0xNoo3** [comment]
> Hey there, theme looks very cozy but there's at least two spots that are pretty bad contrast wise currently.
> 
> Lines with active breakpoints are completely unreadable on all 4 color variants: ![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/739810b7-d99f-41d4-a41e-26acc6187c50)
> 
> Clicking on a highlighted line/variable in the structure view is not completely unreadable, but still bad contrast wise I think: ![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/4527a2f2-8780-4ee5-854c-5cba2396d4cb)
> 
> Otherwise, everything else looks great over here. :)



> Hey there, theme looks very cozy but there's at least two spots that are pretty bad contrast wise currently.
> 
> Lines with active breakpoints are completely unreadable on all 4 color variants: ![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/739810b7-d99f-41d4-a41e-26acc6187c50)
> 
> Clicking on a highlighted line/variable in the structure view is not completely unreadable, but still bad contrast wise I think: ![image](https://github.com/ValentinTT/ida-debugger/assets/4989638/4527a2f2-8780-4ee5-854c-5cba2396d4cb)
> 
> Otherwise, everything else looks great over here. :)



You also had [this](https://github.com/catppuccin/ida-debugger/issues/11) issue while debugging or sync-on?
